health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SetupMicroplan.js

```
@@ -126,6 +126,19 @@ const SetupMicroplan = ({ hierarchyType, hierarchyData }) => {
     };
   }, []);
 
+  useEffect(() => {
+    const handleAssumptionsSubmitEvent = () => {
+      const newKey = parseInt(new URLSearchParams(window.location.search).get("key")) || 1;
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)